HousingManagementSystemApi.Tests/ContollersTests/PropertyEligibleControllerTests.cs

```diff
@@ -36,13 +36,33 @@ private void SetupValidDummyPropertyEligibilityResult()
         [Fact]
         public async Task GivenAPropertyId_WhenAValidRequestIsMade_ItReturnsASuccessfullResponse()
         {
+            // Arrange
             SetupValidDummyPropertyEligibilityResult();
 
+            // Act
             var result = await _systemUnderTest.VerifyPropertyEligibility(_propertyId);
+
+            // Assert
             _verifyPropertyEligibilityUseCase.Verify(x => x.Execute(_propertyId), Times.Once);
             GetStatusCode(result).Should().Be(200);
         }
 
+        [Fact]
+        public async Task GivenAnInvalidPropertyId_ReturnsBadRequest()
+        {
+            // Arrange
+            var invalidPropertyId = "";
```

Comment:
Make it explicitly invalid, or call it blank

Reply:
So change it to null?